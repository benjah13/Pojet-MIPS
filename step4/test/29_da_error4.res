
	Disassembly of section .text

00000000
      00: 00000000	NOP
Invalid param : decimal number awaited in 'vv'
