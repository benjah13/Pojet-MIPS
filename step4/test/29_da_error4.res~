
	Disassembly of section .text

00000000
      00: 00000000	NOP

	End of section .text

Invalid param : decimal number awaited in 'vv'
