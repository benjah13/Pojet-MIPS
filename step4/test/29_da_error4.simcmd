# test da erreur arg2 

da 0x00:1
da 0xae1f:vv
