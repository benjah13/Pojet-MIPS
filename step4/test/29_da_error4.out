00000000:	NOP
Invalid param : decimal number awaited in 'vv'
